src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -129,6 +131,13 @@ public void zeroRotatingArm()
     secondStageMotor2.setSelectedSensorPosition(0, 0, 0);
   }
 
+  /**
+   * Changes the current status of climbMode to th eopposite of what it was
+   */
+  public void setClimbMode() {
+    climbMode = !climbMode;
+  }
```

Comment:
this is a toggle and not a set

Reply:
Also, be sure to still have a set and get method in addition to a toggle